benchmarks/source-agilitycms/README.md

```diff
@@ -0,0 +1,31 @@
+<p align="center">
+  <a href="https://gatsbyjs.org">
+    <img alt="Gatsby" src="https://www.gatsbyjs.org/monogram.svg" height="60" />
+    <img alt="Gatsby" src="https://static.agilitycms.com/layout/img/logo-original.svg" width="200" />
+  </a>
+</p>
+<h1 align="center">
+  Gatsby + Agility CMS : WILL IT BUILD
+</h1>
+
+## To Update Content from the CMS
+
+```shell
+num run update
```

Comment:
Is this a typo? I'm not sure what `nvm` is supposed to be here, but probably not node version manager?

In the package.json I do see a `data-update` so perhaps this was meant to be `npm data-update`?

Also, when I try to run that it says 
> An error occurred while updating the posts list:  TypeError: You must provide either guid or a websiteName.

Not sure how to proceed from there or whether it's meant for consumption outside of Agility CMS-staff at all.

Reply:
It was totally a typo.  If you run `npm run data-update` it will update content in the CMS.